src/ralph/admin/mixins.py

```diff
@@ -195,6 +195,7 @@ def changeform_view(
             extra_context['change_views'] = views
         extra_context['header_obj_name'] = self.model._meta.verbose_name
         self._initialize_search_form(extra_context)
+        extra_context['admin_model'] = self
```

Comment:
i'm not sure if `admin_model` is the best name here. What do you think about `admin_view` or `admin_site`?